public/index.html

```diff
@@ -15,7 +15,7 @@
     <link rel="icon" type="image/png" sizes="48x48" href="/favicons/favicon-48.webp">
   <% } %>
 
-  <meta name="description" content="Superhero is a multi-blockchain wallet to manage crypto assets and navigate the web3 and DeFi space. Powered by æternity.">
+  <meta name="description" content="Superhero is a multi-blockchain wallet to manage crypto assets and navigate the web3 and DeFi space. Supporting Bitcoin, Ethereum and æternity.">
```

Comment:
Please change to "Currently supporting Bitcoin, Ethereum and æternity blockchains."